.gitignore

```diff
@@ -1,3 +1,6 @@
+.vscode/
+!.vscode/settings.json
+
 deployments/*.yml
 node_modules/
 stuff/
```

Comment:
what's stuff?

Reply:
local folder for code snippets and notes